src/index.js

```diff
@@ -28,6 +29,19 @@ function init( {
 				if ( data ) {
 					popup.show( renderPreview( lang, data, isTouch ), target )
 				}
+
+				popup.lang = lang
+				popup.title = title
+				const expanded = root.querySelector( '.wikipediapreview.expanded.mobile' )
+
+				if ( expanded ) {
+					requestPageMedia( lang, title, mediaData => {
+						if ( mediaData && mediaData.length > 0 ) {
```

Comment:
You have `length > 0` here and `length > 1` in `event.js`. That's a good enough reason to make it more DRY.

Reply:
Ha well there you go... do you have a preference for making it available through `utils.js` or `events.js`?

Reply:
This seems to be more in line with the code: https://github.com/wikimedia/wikipedia-preview/pull/21/commits/04ae5e0b6bc26d88b94421bac90b1f95696506ad